manifests/apis/webhooks/resources/namespace.yaml

```diff
@@ -1,4 +1,4 @@
 apiVersion: v1
 kind: Namespace
 metadata:
-  name: rukpak-system
+  name: system
```

Comment:
Why change this name to just `system`? `rukpak-system` was very intuitive to me when I bootstrapped rukpak for the first time. I'd imagine `system` wouldn't be as obvious that that's the namespace where the rukpak resources are deployed.

Reply:
The `namePrefix` in `manifests/apis/webhooks/kustomization.yml` adds `rubpak-` to the names in the webhook/resources.